src/components/InventoryV2/form/useAccessions.ts

```diff
@@ -11,37 +11,46 @@ export type SearchResponseAccession = {
   speciesName: string;
 };
 
-export const useAccessions = (record?: { accessionId?: number }, excludeUsedUp?: boolean) => {
+export const useAccessions = (record?: { accessionId?: number }, speciesId?: number, excludeUsedUp?: boolean) => {
```

Comment:
Can we just use the `speciesId` from the record, if available?

Reply:
I changed this to also use the selected species id if available, so it needs to stay here